### PR TITLE
clean up lcore threading calls

### PR DESCRIPTION
**Pull request by drunkirishcoder**

### CODE REVIEW

File: core/src/runtime/core_map.rs [hunk 1]
```diff
@@ -222,40 +229,42 @@ impl<'a> CoreMapBuilder<'a> {
 
             // spawns a new background thread and initializes a core executor on
             // that thread.
-            let join = thread::spawn(move || {
-                debug!("spawned background thread {:?}.", thread::current().id());
+            let join = thread::Builder::new()
```
Comment (zeeshanlakhani) [inline core/src/runtime/core_map.rs]: this is prob fine, but double-checked formatter here?

File: ffi/src/bindings.h [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
+/// known issues:
+// 1. https://github.com/rust-lang/rust/issues/54341
+
+// all the necessary DPDK functions, types and constants are defined
+// in the following header files.
+#include <rte_eal.h>
+#include <rte_errno.h>
+#include <rte_ethdev.h>
+#include <rte_kni.h>
+#include <rte_lcore.h>
```
Comment (zeeshanlakhani) [inline ffi/src/bindings.h]: we still need the lcore include here? double-checking.

Comment (drunkirishcoder) [reply ffi/src/bindings.h]: no we don't.

File: core/src/runtime/core_map.rs (hunk 1 above)
Comment (drunkirishcoder) [reply core/src/runtime/core_map.rs]: yep ran rustfmt.

**Pull request merged**